corehq/apps/app_manager/static/app_manager/js/details/screen.js

```diff
@@ -71,6 +71,11 @@ hqDefine("app_manager/js/details/screen", function () {
         });
         self.persistCaseContext = ko.observable(detail.persist_case_context || false);
         self.persistentCaseContextXML = ko.observable(detail.persistent_case_context_xml || 'case_name');
+
+        self.caseTileGrouped = ko.observable(detail.case_tile_group.index_identifier || false);
+        self.caseTileGroupBy = ko.observable(detail.case_tile_group.index_identifier);
```

Comment:
What does `detail.case_tile_group.index_identifier` evaluated to if `index_identifier` isn't assigned? Is it `''` or `null`?

Reply:
I think it is `''`